assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -33,13 +35,21 @@ export default {
           };
         },
 
+        @on('didRender')
+        setupReRender() {
+          this.updateHideCategory();
```

Comment:
iirc, it's because without hooking on both events, you create a bug as per my PM communication:  "Unfortunately this property is only set correctly when navigating to the route, not refreshing the route at which point hideCategory seems to be false and the Category renders when it shouldn’t."  Hooking to additional event fixes this.

Reply:
@merefield If you remove the ``setupReRender`` method it seems to work fine?

Reply:
try re-clicking the current view ... it breaks

Reply:
I still cant repro.

Setting:
<img width="738" alt="screen shot 2019-01-08 at 9 34 19 am" src="https://user-images.githubusercontent.com/5931623/50797350-991dd700-1328-11e9-969c-e5aaa0c43756.png">

Steps:

1. Go to latest (see cat column)
2. Go to another list, e.g. top, don't see cat column
3. Toggle back and forth. Cat column behaves as expected.

Reply:
Ahh I see it now

Reply:
Ok, I guess we'll keep it in. It doesn't sit well with me though. There is probably an underlying reason for the issue we're not addressing. Whenever you have multiple observers running the same functions for different stages of the run cycle it almost always means you're patching over some other issue. Let's flag with a code comment for later investigation.